client/main.html

```diff
@@ -6,6 +6,7 @@
 <body>
   {{> topbar }}
   <div class='row' id='container'>
+    {{currentPage}}
     {{> UI.dynamic template=currentPage}}
```

Comment:
OK. THIS RIGHT HERE. Just dropping a standard template in (i.e. `{{> all}}`) works just fine, but I can't figure out how to get Iron router to do the dynamic switching. I tried adding a session.currentpage element thing but not event that is being nice.